Project.toml

```diff
@@ -1,14 +1,14 @@
 name = "LinearFractional"
 uuid = "31851ddc-f9b7-5097-a470-69ef907d7682"
-version = "0.7.6"
+version = "0.7.7"
 
 [deps]
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 
 [compat]
-JuMP = "^1.13"
+JuMP = "=1.22.1"
```

Comment:
This should probably be "1.22.1".  We want at least that version, not exactly that version.

Reply:
oh wait, are you trying to prevent the upgrade?  by capping the upper bound?

Then this should be set as a bound between 1 and 1.22.1 inclusive rather than an exact pin.

Reply:
I updated the compat bounds.